internal/backend/post.go

```diff
@@ -105,8 +104,40 @@ func (d *delphisBackend) CreatePost(ctx context.Context, discussionID string, pa
 	return postObj, nil
 }
 
+func (d *delphisBackend) CreateAlertPost(ctx context.Context, discussionID string, userObj *model.User, isAnonymous bool) (*model.Post, error) {
```

Comment:
We _may_ want to create alert posts for things beyond people joining. I doubt we'd want things like people leaving but imagine a moderator wants to post an alert or we want to post messages into all discussions or something.

Should we update this method to either be more specific `CreateAlertPostForNewParticipant` or generalize it?

Reply:
do you know what other types we'd like? once I have a different one, I can more easily generalize this one